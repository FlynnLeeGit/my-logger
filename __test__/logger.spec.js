const { SimpleLogger } = require('../index')

describe('SimpleLogger basic', () => {
  test('should exist', () => {
    expect(SimpleLogger).toBeDefined()
  })
  test('should contractor ok', () => {
    const sl = new SimpleLogger({
      scope: 'test',
      theme: 'yellow'
    })
    expect(sl.title).toMatchSnapshot()
  })
  test('json should output inspect formate', () => {
    const sl = new SimpleLogger()
    sl.log({ a: 1 })
  })
  test('should output ok', () => {
    const sl = new SimpleLogger()
    sl.log(1)
    sl.warn(1)
    sl.error(1)
    sl.info(1)
  })
})
