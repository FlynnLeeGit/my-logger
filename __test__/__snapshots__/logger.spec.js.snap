// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleLogger basic should contractor ok 1`] = `"[33mtest[39m"`;
